gui/cefdisplay/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+############################################################################
+# CMakeLists.txt file for building ROOT gui/cefdisplay package
+############################################################################
+
+set(libname ROOTCefDisplay)
+
+ROOT_GLOB_SOURCES(CEF_sources src/simple_app.cxx src/base_handler.cxx src/osr_handler.cxx src/gui_handler.cxx)
```

Comment:
No need to use `ROOT_GLOB_SOURCES()` when you list all files, you can just use `set(CEF_SOURCES src/simple_app.cxx ...)`